### PR TITLE
Fix bob widget issues

### PR DESCRIPTION
**Pull request by abigailalexander**
This PR fixes various issues with widgets that I noticed when using .bob files:

- Added `image_stretch` field (this is `stretch_to_fit` in .opi files)
- Modified bobParseFonts to use strings instead of numbers as keys
- Added a check for whether a .opi or .bob file is used for rules, as .opi uses `pv_name` and .bob uses `pv`
- Changed default background and text colour for Label and Readback to match with Phoebus defaults

### CODE REVIEW

File: src/ui/widgets/EmbeddedDisplay/bobParser.ts [hunk 1]
```diff
@@ -245,10 +245,10 @@ export function parseBob(
       (pvName: ElementCompact): PV => opiParsePvName(pvName, defaultProtocol)
     ],
     font: ["font", bobParseFont],
-    rules: [
-      "rules",
-      (rules: Rule[]): Rule[] => opiParseRules(rules, defaultProtocol)
-    ],
+    // rules: [
```
Comment (rjwills28) [inline src/ui/widgets/EmbeddedDisplay/bobParser.ts:248]: Can we remove this commented out code now?

File: src/ui/widgets/Readback/readback.module.css [hunk 2]
```diff
@@ -5,11 +5,11 @@
   white-space: nowrap;
   text-overflow: ellipsis;
   background-color: #383838;
```
Comment (rjwills28) [inline src/ui/widgets/Readback/readback.module.css:7]: The background colour for the readback widget is currently very dark, whereas the default Phoebus background is a lighter grey - I believe the `background-color` should be `#f0f0f0`.
Current: 
current_readback
Proposed:
readback_new

**Review by rjwills28 — COMMENTED** [review]
Thanks for these changes. I've just added a few comments/suggestions.

File: src/ui/widgets/EmbeddedDisplay/bobParser.ts (hunk 1 above)
Comment (abigailalexander) [reply src/ui/widgets/EmbeddedDisplay/bobParser.ts:248]: Ah yes, I meant to remove that. Thank you for spotting it

File: src/ui/widgets/Readback/readback.module.css (hunk 2 above)
Comment (abigailalexander) [reply src/ui/widgets/Readback/readback.module.css]: Thanks for the spot, changed this now.

**Review by rjwills28 — APPROVED** [review]

**Pull request merged**